core/src/main/java/com/netease/arctic/table/TableProperties.java

```diff
@@ -213,7 +213,8 @@ private TableProperties() {}
   public static final int AUTO_CREATE_TAG_MAX_DELAY_MINUTES_DEFAULT = 60;
 
   public static final String AUTO_CREATE_TAG_DAILY_FORMAT = "tag.auto-create.daily.tag-format";
```

Comment:
How about add a new hourly format property?
```java
public static final String AUTO_CREATE_TAG_HOURLY_FORMAT = "tag.auto-create.hourly.tag-format";
public static final String AUTO_CREATE_TAG_HOURLY_FORMAT_DEFAULT = "'tag-'yyyyMMddHH";
```

Reply:
An alternative solution is to remove "daily" from the property name, then it will be 
```java
public static final String AUTO_CREATE_TAG_FORMAT = "tag.auto-create.tag-format";
public static final String AUTO_CREATE_TAG_FORMAT_DAILY_DEFAULT = "'tag-'yyyyMMdd";
public static final String AUTO_CREATE_TAG_FORMAT_HOURLY_DEFAULT = "'tag-'yyyyMMddHH";
```
I think both are acceptable, which one do you prefer?

Reply:
> An alternative solution is to remove "daily" from the property name, then it will be
> 
> ```java
> public static final String AUTO_CREATE_TAG_FORMAT = "tag.auto-create.tag-format";
> public static final String AUTO_CREATE_TAG_FORMAT_DAILY_DEFAULT = "'tag-'yyyyMMdd";
> public static final String AUTO_CREATE_TAG_FORMAT_HOURLY_DEFAULT = "'tag-'yyyyMMddHH";
> ```
> 
> I think both are acceptable, which one do you prefer?

There may be a problem here. For example, `hourly` is configured but the user passes 'tag-'yyyyMMdd.
 Should the parameters tag.auto-create.tag-format-prefix and tag.auto-create.tag-format-suffix be used directly here?  WDYT?

Reply:
Users may also have costom needs in terms of time format(like `yyyy-MM-dd`).
Another case, when using `hourly` tag with a  offset like 30min, it is also reasonable to set the format accurate to the minute, like `tag-20230102-12:30`.
So only `prefix` and `suffix` can't meet the above requirements, and 3 parts are needed for this format configuration (`prefix`,`time-format`,`suffix`), which is too complicated.

Reply:
> There may be a problem here. For example, hourly is configured but the user passes 'tag-'yyyyMMdd.

This is a situation of configuration error, which may lead to abnormal situations, such as the inability to create tags.  We cannot prevent user's misconfigurations.

Reply:
> > There may be a problem here. For example, hourly is configured but the user passes 'tag-'yyyyMMdd.
> 
> This is a situation of configuration error, which may lead to abnormal situations, such as the inability to create tags. We cannot prevent user's misconfigurations, and I believe this falls outside the scope of the issues we should address.

ok, Regarding the two configuration methods, I prefer to use the second one. The configuration will be a less compared to the first one.